### DISCUSSION TITLE
How to use the .db3？

### DISCUSSION THREAD

**Discussion by XinZhaoDC** [DapengFeng/S3E#21] [Q&A] [answered]
Thank you for opening dataset. However, I didn't find the instruction for using the *.db3 data in your dataset.

**Comment by DapengFeng** [comment, marked as answer]
*.db3 is the ros2 bag format, you can play back the data following the [ros2 intruction](https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Recording-And-Playing-Back-Data/Recording-And-Playing-Back-Data.html).

If you are non ros user, we provide the [rosbag_extractor](https://github.com/thisparticle/rosbag_extractor) to extract the images, 3d point cloud and orther data fields from the ros bag.

**Reply by XinZhaoDC** [reply]
Got it. Thank you for reply!
